dist/profile/manifests/datadog_ssl_check.pp

```diff
@@ -0,0 +1,29 @@
+#
+# Define datadog check for ssl expiration
+# Inspired by: https://workshop.avatarnewyork.com/project/datadog-ssl-expires-check/
+#
+
+class profile::datadog_ssl_check (
+  $sites = undef,
```

Comment:
I am not sure to understand your point, I don't see any error if the value is define but I would llike to have a warning if the value is not defined.